apps/src/applab/designMode.js

```diff
@@ -383,6 +385,8 @@ designMode.updateProperty = function(
 
       if (ICON_PREFIX_REGEX.test(value)) {
         element.src = assetPrefix.renderIconToString(value, element);
+      } else if (DATA_PREFIX_REGEX.test(value)) {
+        element.src = assetPrefix.fixPath(value);
```

Comment:
Question: Can you give a little context on why you added fixPath here?

Reply:
Not needed. Removed in latest commit.